src/components/markdown/markdown.tsx

```diff
@@ -32,11 +33,15 @@ export class Markdown {
     @Prop()
     public value: string;
 
+    @Prop()
+    public whitelist?: CustomComponent[];
+
```

Comment:
Don't forget to add an example, showcasing this new prop in the docs

Reply:
done!